bsp/boards/openmote-b/board.c

```diff
@@ -47,20 +46,22 @@ bool board_timer_expired(uint32_t future);
 static void clock_init(void);
 static void gpio_init(void);
 static void button_init(void);
-static void antenna_init(void);
 
+static void antenna_init(void);
```

Comment:
there are 2 antennas, why these functions?!?

Reply:
I am not really sure. My understanding is the they define which radio chip uses the 24ghz antenna

Reply:
One antenna is 2.4 GHz only, the other one is 868 MHz only. And there are 3 radios in total, two at 2.4 GHz and one at 868 MHz. On the OpenMoteB schematic, I see U8 (bottom right), which looks like an RF switch. It switches the 2.4 GHz antenna between the cc2538 2.4 GHz radio port and the at86rf215 2.4 GHz radio port. PD3 and PD4 of the cc2538 look like the control signals of this RF switch. I'm guessing that the functions discussed here activate these PD3 and PD4 signals.